ori/volume/server/volume_list.go

```diff
@@ -148,9 +53,29 @@ func (s *Server) filterVolumes(volumes []*ori.Volume, filter *ori.VolumeFilter)
 	return res
 }
 
+func (s *Server) listVolumes(ctx context.Context, log logr.Logger) ([]*ori.Volume, error) {
+	cephImages, err := s.imageStore.List(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("error listing volumes: %w", err)
+	}
+
+	var res []*ori.Volume
+	for _, cephImage := range cephImages {
+		oriVolume, err := s.convertImageToOriVolume(ctx, log, cephImage)
```

Comment:
If `imageStore.List` returns a volume that is not managed by the `Server`, the whole `List` operation errors. `listVolumes` should filter out volumes that are not created / managed via `ori` (for reference, e.g. the `machinebroker` filters by managed and created `Machine`s in `onmetal-api`).

Reply:
`imageStore.List(ctx)` will only return the content of the `omap` hence it is still possible to create `ceph images` directly without interfering here. Since a `VolumePool` is corresponding to one `ceph pool` two instances also will not conflict (since omaps are not shared across pools).

Reply:
But what if I create an `api.Image` using the internal API only (i.e. `imageStore.Create`) and this `api.Image`. This `api.Image` was not created via `ori` and should not appear in the result set of `listVolumes` but currently it would. Even worse, as far as I can see it'd cause an error since using the internal API, I'm not obliged to specify the `volumeClassLabel`, resulting in an error upon calling `convertImageToOriVolume`.